powerlevel9k.zsh-theme

```diff
@@ -1006,10 +1006,16 @@ prompt_ssh() {
 # Status: return code if verbose, otherwise just an icon if an error occurred
 set_default POWERLEVEL9K_STATUS_VERBOSE true
 set_default POWERLEVEL9K_STATUS_OK_IN_NON_VERBOSE false
+set_default POWERLEVEL9K_STATUS_NICE_EXIT_CODE false
 prompt_status() {
   if [[ "$RETVAL" -ne 0 ]]; then
     if [[ "$POWERLEVEL9K_STATUS_VERBOSE" == true ]]; then
-      "$1_prompt_segment" "$0_ERROR" "$2" "red" "226" "$RETVAL" 'CARRIAGE_RETURN_ICON'
+      if [[ "$POWERLEVEL9K_STATUS_NICE_EXIT_CODE" == true ]]; then
+        "$1_prompt_segment" "$0_ERROR" "$2" "red" "226" \
+          "$(nice_exit_code "$RETVAL" | awk -F ':' '{print $2 "(" $1 ")"}')" 'CARRIAGE_RETURN_ICON'
```

Comment:
Nice catches, @docwhat, on `nice_exit_code` and the conflict.

@sei40kr - Once you rebase your branch onto `next` you'll see the conflict.